From 40836ce7c724491f5e605bc53153edc844d1119b Mon Sep 17 00:00:00 2001
From: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Date: Tue, 17 Nov 2020 08:31:12 +0000
Subject: [PATCH] arm64: dts: rockchip: rk3568: fix rkcif_mmu error

error log:
[  352.947003] rkcif_dvp: Allocate dummy buffer, size: 0x00075300
[  352.947160] rk_iommu fdfe0800.iommu: Error during raw reset.
MMU_DTE_ADDR is not functioning
[  352.947236] rkcif_dvp: Failed to get runtime pm, -14

Signed-off-by: Allon Huang <allon.huang@rock-chips.com>
Signed-off-by: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Change-Id: I1eab7daed218a40e2d6f14499e1e24da927953e5
---
 arch/arm64/boot/dts/rockchip/rk3568.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568.dtsi b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
index 0cde1070bc32..8ee9ff911c3a 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568.dtsi
@@ -1104,6 +1104,7 @@
 		clock-names = "aclk", "iface";
 		power-domains = <&power RK3568_PD_VI>;
 		#iommu-cells = <0>;
+		rockchip,disable-mmu-reset;
 		status = "disabled";
 	};
 
-- 
2.35.3

